website/docs/d/stream_connection.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Do we need to add a new example in the example folder?

Reply:
will do